content/20.results.md

```diff
@@ -7,130 +7,100 @@ Keynote speakers for RECOMB were available for all years from 1997--2019.
 We included individuals who were honored multiple times as separate entries.
 For example, Christine Orengo was a keynote speaker at RECOMB 2004 and became an ISCB Fellow in 2016, and thus was counted twice in this list.
 
-We sought to compare this dataset with a background distribution of potential speakers, which we considered to be last or senior authors of bioinformatics and computational biology manuscripts.
-We used those published in [Bioinformatics](https://academic.oup.com/bioinformatics), [BMC Bioinformatics](https://bmcbioinformatics.biomedcentral.com/), and [PLOS Computational Biology](https://journals.plos.org/ploscompbiol/) as a set of bioinformatics and computational biology manuscripts.
-We downloaded the metadata of manuscripts published in these journals from PubMed, which provided almost 30,000 articles for evaluation.
-However, although PubMed provides author order, it does not provide corresponding author information.
-To determine corresponding authors for an article, we used the PMC corresponding author information when it was available (20,696 articles) and the PubMed last author as a fallback when corresponding author information was missing (9,053 articles).
+We sought to compare this dataset with a background distribution of potential speakers, which we considered to be last authors of bioinformatics and computational biology manuscripts.
+We scraped PubMed for manuscripts written in English from 1993--2019 with the MeSH term ["computational biology"](https://pubmed.ncbi.nlm.nih.gov/?term=%22Computational+Biology%22%5BMeSH+Terms%5D+).
+We downloaded the metadata of manuscripts published in these journals from PubMed, which provided 173,735 articles for evaluation.
+For each article, we extracted its last author's fore name and last name for analysis.
 
-### Similar gender proportion between ISCB's honorees and the field
+In a [previous version](https://greenelab.github.io/iscb-diversity-manuscript/v/711edb01f5335a4c1bb8e1e0575618586a7abf9b/) of this work, instead of last authors, we examined corresponding authors of articles in three well-recognized computational biology journals, and the results were consistent with our findings here.
```

Comment:
cite versioned preprint here on biorxiv with URL (since their DOI is not versioned)?